bucket/okular.json

```
@@ -0,0 +1,34 @@
+{
+    "version": "20.08.3-478",
+    "description": "Universal document viewer",
+    "homepage": "https://okular.kde.org",
+    "license": "LGPL-2.0-only",
+    "architecture": {
+        "64bit": {
+            "url": "https://binary-factory.kde.org/view/Windows%2064-bit/job/Okular_Release_win64/478/artifact/okular-20.08.3-478-windows-msvc2019_64-cl.7z",
+            "hash": "cfe836b3c8d74a9406950e747763224b8a0274f3096b24d0cc89c175f2c3d705"
+        }
+    },
+    "bin": "bin\\okular.exe",
+    "shortcuts": [
+        [
+            "bin\\okular.exe",
+            "okular"
+        ]
+    ],
+    "checkver": {
+        "url": "https://binary-factory.kde.org/view/Windows%2064-bit/job/Okular_Release_win64/lastSuccessfulBuild/artifact/",
+        "regex": "okular-([\\d.-]+)-windows-(?<lib>\\w+)-cl\\.7z"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://binary-factory.kde.org/view/Windows%2064-bit/job/Okular_Release_win64/$preReleaseVersion/artifact/okular-$matchHead-$preReleaseVersion-windows-$matchLib-cl.7z"
```

- Why $matchHead with combination of prerelease? Completely nonsense

```suggestion
                "url": "https://binary-factory.kde.org/view/Windows%2064-bit/job/Okular_Release_win64/$preReleaseVersion/artifact/okular-$version-windows-msvc2019_64-cl.7z"
```